src/Aws/Common/Credentials/Credentials.php

```diff
@@ -147,7 +147,7 @@ public static function fromIni($profile = null, $filename = null)
             throw new \RuntimeException("Invalid AWS credentials file: {$filename}.");
```

Comment:
Yeah, that makes sense. That way, the next block of code throws the exception, which is a more accurate message for that case. I'll update the PR.